tradesman/model.py

```diff
@@ -138,6 +140,10 @@ def import_buildings(self):
 
         building_import(self.__model_place, self._project, self.__osm_data)
 
+    def create_synthetic_population(self):
```

Comment:
Let's divide this into two parts:

1. Creation of the data structure in disk
2. Running of PopSim

This allows the user to inspect the files (and maybe change the seeds) before creating the synthetic population